w3f-ring-proof/src/piop/mod.rs

```diff
@@ -125,7 +126,7 @@ impl<F: PrimeField, G: AffineRepr<BaseField = F>> FixedColumns<F, G> {
     }
 }
 
-#[derive(CanonicalSerialize, CanonicalDeserialize)]
+// #[derive(CanonicalSerialize, CanonicalDeserialize)]
```

Comment:
@davxy hdyt, anyone uses this serialization? I don't think it's useful as is, as the ring might have changed, and it's fast to recompute.